src/ContaoCommunityAlliance/DcGeneral/Contao/View/Contao2BackendView/EditMask.php

```diff
@@ -713,4 +716,63 @@ class_implements(
 
         return $objTemplate->parse();
     }
+
+    /**
+     * Get the breadcrumb navigation via event.
+     *
+     * @return string
+     *
+     * @SuppressWarnings(PHPMD.Superglobals)
+     * @SuppressWarnings(PHPMD.CamelCaseVariableName)
+     */
+    protected function breadcrumb()
+    {
+        $event = new GetBreadcrumbEvent($this->getEnvironment());
+
+        $this->getEnvironment()->getEventDispatcher()->dispatch(sprintf('%s', $event::NAME), $event);
+
+        $arrReturn = $event->getElements();
+
+        if (!is_array($arrReturn) || count($arrReturn) == 0) {
+            return null;
+        }
+
+        $GLOBALS['TL_CSS'][] = 'system/modules/dc-general/html/css/generalBreadcrumb.css';
+
+        $objTemplate = $this->getTemplate('dcbe_general_breadcrumb');
+        $this->addToTemplate('elements', $arrReturn, $objTemplate);
```

Comment:
Could you use `$template->set(/* .... */);` here?
The `addToTemplate()` calls are from a time where there was no setter method in the template yet and therefore anonymous properties on the template were set which we dislike.